data/json/recipes/recipe_weapon.json

```diff
@@ -1135,6 +1135,20 @@
     ],
     "components": [ [ [ "pipe", 3 ] ], [ [ "nail", 3 ] ], [ [ "scrap", 3 ] ], [ [ "2x4", 1 ] ] ]
   },
+  {
+    "type": "recipe",
+    "result": "flamethrower_simple",
+    "category": "CC_WEAPON",
+    "subcategory": "CSC_WEAPON_RANGED",
+    "skill_used": "mechanics",
+    "skills_required": [ "gun", 3 ],
+    "difficulty": 6,
```

Comment:
This recipe shouldn't have a marksmanship requirement and mechanically it's simple enough that it shouldn't need mechanics skill over 3.